app-modules/engagement/src/Filament/Actions/Concerns/ImplementsHasBulkEngagementAction.php

```diff
@@ -61,13 +61,14 @@ public function cancelBulkEngagementAction(): Action
                 $this->engageActionData = [];
                 $this->engageActionRecords = [];
             })
-            ->modalDescription(fn () => 'The message has not been sent, are you sure you wish to return to the list view?')
+            ->modalHeading('Are you sure you wish to cancel?')
+            ->modalDescription(fn () => 'If you cancel, the message will not be sent.')
```

Comment:
Does this need to be a function? I think you may just be able to put the text here.

Reply:
function removed.